tests/unit/test_space.py

```diff
@@ -370,11 +407,31 @@ def test_box_contains_point(point: tf.Tensor) -> None:
 )
 def test_box_does_not_contain_point(point: tf.Tensor) -> None:
     assert point not in Box(tf.constant([-1.0, 0.0, -2.0]), tf.constant([2.0, 1.0, -0.5]))
+    assert not Box(tf.constant([-1.0, 0.0, -2.0]), tf.constant([2.0, 1.0, -0.5])).contains(point)
+
+
+@pytest.mark.parametrize(
+    "points, contains",
+    [
+        (tf.constant([[-1.0, 0.0, -2.0], [-1.1, 0.0, -2.0]]), tf.constant([True, False])),
+        (tf.constant([[[0.5, 0.5, -1.5]]]), tf.constant([[True]])),
+    ],
+)
```

Comment:
Is there a reason to pass the points and contains values in using `parametrize` but not to pass the box in?